lib/logdna.rb

```diff
@@ -60,32 +60,26 @@ def level=(value)
     end
 
     def log(message = nil, opts = {})
-      if message.nil? && block_given?
-        message = yield
-      end
+      message = yield if message.nil? && block_given?
       if message.nil?
         puts "provide either a message or block"
         return
       end
       message = message.to_s.encode("UTF-8")
-      @client.write_to_buffer(message, default_opts.merge(opts).merge(
-                                         timestamp: (Time.now.to_f * 1000).to_i
-                                       ))
+      @client.write_to_buffer(message, default_opts.merge(opts).merge(timestamp: (Time.now.to_f * 1000).to_i))
     end
 
     Resources::LOG_LEVELS.each do |lvl|
       name = lvl.downcase
 
       define_method name do |msg = nil, opts = {}, &block|
-        self.log(msg, opts.merge(
-                        level: lvl
-                      ), &block)
+        log(msg, opts.merge(level: lvl), &block)
```

Comment:
revert self here.

Reply:
Again, @vilyapilya, by `Rubocop` - @vilyapilya, before publishing, we'll have a chance to do integration and regression tests - in order to have it in a way you prefer, we'll need to update `.rubocop.yml`

Reply:
I have this file updated in my PR.